build_resolvers/lib/src/resolver.dart

```diff
@@ -256,7 +256,7 @@ class AnalyzerResolver implements ReleasableResolver {
   Future<List<ErrorsResult>> _syntacticErrorsFor(LibraryElement element) async {
     final existingElements = [
       element,
-      for (final part in element.parts)
+      for (final part in element.parts2)
```

Comment:
It looks like this no longer gives the actual part source, but just the library source.

How can I get the part source? It seems hidden now.

Reply:
Should I cast these to a `DirectiveUriWithSource`? Or just do a check for that type?

Reply:
Ok yeah that looks like the fix, I updated to check for the `DirectiveUriWithSource` type and then use the source from that. LMK if I should do something else.